.idea/IAA.iml

```diff
@@ -5,6 +5,7 @@
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/api/target" isTestSource="false" generated="true" />
       <sourceFolder url="file://$MODULE_DIR$/frontend/src/app/api" isTestSource="false" generated="true" />
+      <sourceFolder url="file://$MODULE_DIR$/backend/src/main/java/de/nordakademie/iaa/noodle/api" isTestSource="false" generated="true" />
```

Comment:
What exactly is this supposed to do? I don't see any folder specifically marked in IntelliJ because of this.

Reply:
Thats the folder which will be autogenerated when creating the api interfaces:
![Screen Shot 2020-10-27 at 17 14 28](https://user-images.githubusercontent.com/7913185/97329940-e52c6980-1877-11eb-8a9f-b27b64029911.png)

Reply:
Yeah I saw that. I just don't know what the entry in the .iml does. Seems to me that only the .gitignore causes the mark itself. Anyway, doesn't do harm either, I guess.